### ISSUE TITLE
example does not render different pages

### ISSUE THREAD

**Issue by mgenev** [tanepiper/quorra#4]
when I run the example and browse around with the router, I always get this from app.jsx

`This example shows how you can create an isomorphic app with Hapi and React-Router`

only the heading above it changes as `state.message` changes:
`<h1>{this.state.message}</h1>`

I never get the markup on the other pages like:
`<div>This is the about page</div>` even when the `state.message` says i'm on the about page.

**Comment by j** [comment]
Yep, I came to check out this project thinking, "oh, awesome". I cloned, ran the example, and :hankey:.  No bueno!

**Comment by willin** [comment]
15.0.1 not support?
